src/test/java/org/folio/innreach/repository/CentralServerRepositoryTest.java

```diff
@@ -17,7 +18,8 @@
 import org.folio.innreach.fixture.LocalServerCredentialsFixture;
 
 
-@Sql(scripts = "classpath:db/central-server/pre-populate-central-server.sql")
+@Sql(scripts = {"classpath:db/central-server/pre-populate-central-server.sql",
+  "classpath:db/itm-contrib-opt-conf/pre-populate-itm-contrib-opt-conf.sql"})
```

Comment:
Same here as for the controller. Not sure that item contribution options should affect Central Server repository